features/src/main/scala/com/salesforce/op/features/types/Maps.scala

```diff
@@ -305,6 +305,14 @@ object NameStats {
     val LastName = "lastName"
     val Gender = "gender"
   }
+  // TODO: Use enumeratum for this
```

Comment:
so why not use Enumerantum indeed? @MWYang

Reply:
Oops, just forgot to do it 😅